styles/mixins/_focus.scss

```diff
@@ -2,6 +2,11 @@
 
 $-color-focus-outline: var.$color-focus-outline;
 
+@mixin outline-rule($inset: false) {
```

Comment:
I couldn't use the mixins as written because I needed to craft a specific selector. Here I'm externalizing at least part of the rule-set so I can use that.